src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateServiceTest.kt

```diff
@@ -43,15 +44,16 @@ class AssessmentUpdateServiceTest {
     courtCaseRestClient,
     assessmentUpdateRestClient,
     offenderService,
-    assessmentSchemaService
+    assessmentSchemaService,
   )
   private val assessmentUpdateService = AssessmentUpdateService(
```

Comment:
This is getting a bit too large, we should look at refactoring it to make it more concise

Reply:
💯  - there's quite a few dependencies being injected to this service, this comment in the tests says it all... `// Christ, what a lot of set up` 😅